src/js/common/uuidGeneratorHelper.js

```diff
@@ -0,0 +1,5 @@
+import uuid from "uuid";
+
+export const getDeviceId = () => {
```

Comment:
Please change this to be like this:

`export const getDeviceId = () => uuid();`

Reply:
done